packages/grid/_modules_/grid/components/column-headers.tsx

```diff
@@ -95,11 +95,11 @@ export const ColumnsHeader = React.memo(
     return (
       <React.Fragment>
         <ScrollArea scrollDirection="left" />
+        {/* eslint-disable-next-line */}
```

Comment:
I would only ignore this specific rule and explain why it doesn't apply

```suggestion
        {/* row isn't interactive, cells are, event delegation */}
        {/* eslint-disable-next-line jsx-a11y/interactive-supports-focus */}
```